woocommerce/woocommerce-ios
# Updates help screen to point at Woo docs site instead of Woo FAQ

Closes #836 

Instead of pointing the user to a general WooCommerce FAQ in the help screen we now point them at the docs section for the iOS app specifically.

1. Changes `Browse our FAQ` help item to `Help Center`. If we ever move to Zendesk's Help Center this text doesn't have to change then.
2. Renames the Tracks event being fired when someone views the documentation.
3. Reorders a folder of files alphabetically because it made me stabby.

# Testing

1. Launch the app in a logged-out state.
2. Verify help screens are as expected.
3. Tap the Help Center row and validate the iOS documentation is showing.
4. Log into the app and validate 2 & 3 are still correct.
